userapp/src/components/table/Table.js

```diff
@@ -1,15 +1,99 @@
-import React from 'react'
-import TableHead from './TableHead'
-import TableBody from './TableBody'
-
-
-const Table = () => {
-    return (
-        <div>
-            <TableHead />
-            <TableBody />
-        </div>
-        )
+import React from "react"
+import { useTable } from "react-table"
```

Comment:
is this react strap? We should have our tables in MUI right?

Reply:
No reactstrap